app/controllers/User/registrations_controller.rb

```diff
@@ -1,6 +1,7 @@
 class User::RegistrationsController < Devise::RegistrationsController
```

Comment:
In your case you don't need any of the Devise controllers. Typically you only need them when doing unorthodox things. Since they're all commented out, you can just delete the files.